decidim-conferences/config/locales/en.yml

```
@@ -288,7 +288,23 @@ en:
           error: There was a problem updating a registration type for this conference.
           success: Conference registration type successfully updated.
       titles:
+        conference_admins: Conference admins
+        conference_invites: Conference invites
+        conference_partners: Conference partners
```

For consistency with the views' title:
```suggestion
        conference_partners: Partners
```